man/fi_msg.3.md

```diff
@@ -138,12 +138,12 @@ per call through the use of flags.  The fi_sendmsg function takes a
 
 {% highlight c %}
 struct fi_msg {
-	const struct iovec *msg_iov;/* scatter-gather array */
-	void               **desc;  /* local request descriptor */
-	size_t             iov_count;/* # elements in iov */
-	const void         *addr;   /* optional endpoint address */
-	void               *context;/* user-defined context */
-	uint64_t           data;    /* optional message data */
+        const struct iovec *msg_iov;  /* scatter-gather array */
+        void               **desc;    /* local request descriptors */
+        size_t             iov_count; /* # elements in iov */
+        fi_addr_t          addr;      /* optional endpoint address */
```

Comment:
The formatting of this line differs from those around it.  Why was the formatting of the other lines changed?